src/main/java/com/hz6826/clockin/init/DatabaseInit.java

```diff
@@ -3,52 +3,192 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
-import java.util.HashMap;
+import java.util.ArrayList;
 import java.util.Scanner;
 
-import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonArray;
+import java.util.Map;
+import java.util.Set;
 
 import com.hz6826.clockin.ClockIn;
-import org.jetbrains.annotations.Nullable;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.DatabaseMetaData;
+import java.sql.Statement;
+
+
+/*
+ * 储存了sql的信息
+ * */
+class sql_list {
+    public String dataBaseName;
+    public JsonObject sql_tables;
+
+    public sql_list(String dataBaseName, JsonObject sql_tables) {
+        this.sql_tables = sql_tables;
+        this.dataBaseName = dataBaseName;
+    }
+};
 
 public class DatabaseInit {
 
-    private HashMap<String,String> sql;
+    private sql_list sql = null;
+    private Connection connection;
+    private String DataBaseName;
+
+    /*
+     *   构造器初始化sql表,并根据数据库类型建立表
+     * */
+    public DatabaseInit(Connection DBconnection) {
```

Comment:
```suggestion
    public static void DatabaseInit(Connection DBconnection) {
```
实用工具类就够了，不用弄对象，全用静态方法